### DISCUSSION TITLE
Question about PFlow and TDS

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#290] [Internal Design] [answered]
Hi Hantao,

I have a question about the data transferring about routine PFlow and TDS.
I altered the `StaticGen` setpoints by `model.alter()`, although I ran PFlow before the TDS, the `GENROU` setpoints (`Pe`, `Qe`) remain unchanged.
Do you have any suggestions?

Regards,
Jinning

**Comment by cuihantao** [comment, marked as answer]
The alteration of StaticGen’s initial values need to be done before solving the power flow. Once power flow is solved and TDS is initialized, alteration in StaticGen will not propagate to dynamic models.

**Reply by jinningwang** [reply]
Thanks for your reply!
